Cabal/src/Distribution/Simple/Configure.hs

```diff
@@ -2499,7 +2499,7 @@ checkForeignDeps pkg lbi verbosity =
         explainErrors missingHdr missingLibs
     )
   where
-    allHeaders = collectField (fmap getSymbolicPath . includes)
+    allHeaders = []
```

Comment:
This area of the code checked that a dummy C program that includes any headers listed under `includes` can be compiled successfully.